### PR TITLE
Add github-actions.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

- [x] Fix static analysis.
- [x] Add github-actions.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+
+# Ensure those won't be messed up with
+*.png  binary
+*.jpg  binary
+*.gif  binary
+*.ttf  binary
+
+# Ignore some meta files when creating an archive of this repository
+# We do not ignore any content, because this repo represents the
+# `yiisoft/yii2-dev` package, which is expected to ship all tests and docs.
```
Comment (samdark) [inline .gitattributes:26]: This note is from Yii 2.

File: README.md [hunk 2]
```diff
@@ -11,18 +11,21 @@ to events dispatched.
...
-[![Build Status](https://travis-ci.com/yiisoft/event-dispatcher.svg?branch=master)](https://travis-ci.com/yiisoft/event-dispatcher)
+[![Build Status](https://github.com/yiisoft/event-dispatcher/workflows/build/badge.svg)](https://github.com/yiisoft/event-dispatcher/actions?query=workflow%3Abuild)
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/event-dispatcher/badges/coverage.png)](https://scrutinizer-ci.com/g/yiisoft/event-dispatcher/)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/event-dispatcher/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/event-dispatcher/?branch=master)
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https://badge-api.stryker-mutator.io/github.com/yiisoft/event-dispatcher/master)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/event-dispatcher/master)
+[![static analysis](https://github.com/yiisoft/event-dispatcher/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/event-dispatcher/actions?query=workflow%3A%22static+analysis%22)
 
-## Features
+
+### Features
```
Comment (samdark) [inline README.md:21]: Why h3?

File: composer.json [hunk 3]
```diff
@@ -16,12 +16,17 @@
...
     "require": {
         "php": "^7.4",
         "psr/event-dispatcher": "1.0.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.0"
+        "infection/infection": "^0.16.3",
+        "phan/phan": "^3.0.2",
+        "phpunit/phpunit": "^9.2",
+        "yiisoft/di": "^3.0@dev"
```
Comment (samdark) [inline composer.json:29]: If we want to have EventDispatcherProvider, we have to more this dependency to `require`.

File: .gitattributes (hunk 1 above)
Comment (terabytesoftw) [reply .gitattributes:26]: done.

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md:21]: The test notes are in H3, all the repositories will be the same.

File: composer.json (hunk 3 above)
Comment (terabytesoftw) [reply composer.json:29]: done.

**Comment by samdark** [comment]
👍

**Pull request merged**